lib/Test2/Harness/Renderer/JUnit.pm

```diff
@@ -49,7 +51,13 @@ sub render_event {
     my $f      = $event->{facet_data};
     my $job    = $f->{harness_job};
     my $job_id = $event->{'job_id'} or return;
-    my $stamp  = $event->{'stamp'} || die "No time stamp found for event?!?!?!?";
+
+    my $stamp;
+    if ( $f->{'harness_job_end'} ) {
+        $stamp = $f->{'harness_job_end'}->{'times'}->{'source'}->{'stop'} || die Dumper($event) . "No time stamp found for event?!?!?!?";
```

Comment:
I'm not wild about this change. This seems to be working around an upstream bug found here https://github.com/Test-More/Test2-Harness/issues/117 . 

In my own testing, I found using inconsistent stamps caused the test times to be significantly inaccurate (sometimes negative run times). 

@exodist has already agreed that this is a bug that this event doesn't have a time stamp. Could we possibly always only fall back if ```$event->{'stamp'}``` isn't available since that's what's supposed to be present in all events?

Reply:
I force pushed a fix for this. It now checks the event first in all cases.